app/screens/TripHome.tsx

```diff
@@ -1,26 +1,99 @@
 import React, { FC } from "react"
 import { observer } from "mobx-react-lite"
-import { Button, Screen, Text, useAutoImage } from "../components"
+import { Button, Icon, Screen, Text, useAutoImage } from "../components"
 import { AppStackScreenProps } from "../navigators"
 import { ScrollView, View } from "react-native"
 import { useAuth } from "../models/hooks"
+import TripCard from "../components/TripCard"
 
 type TripHomeProps = AppStackScreenProps<"TripHome">
 export const TripHome: FC<TripHomeProps> = observer(function TripHomeScreen() {
   const { doLogout } = useAuth()
+  const background = {
+    back: require("../../assets/images/gradientBg.png"),
+  }
+  const exTrips = [
```

Comment:
I would rename these trips, and when u display the trips below, do trips.map() and set up a function to separate current and upcoming trips so when we connect this to the data response from backend ( which will come in chronological order ) we just need to set this variable up to store the fetch response.